### PR TITLE
UI Makeover

### PR DESCRIPTION
**Pull request by Polygonalr**
Going with a purple theme to keep things consistent with our logo. Taking inspiration from [Material Design](https://material.io/design). Light theme will be the default _(read my Github Bio)_, but the dark theme stylesheet will still exist if we want to add theme toggling as a potential v1.2b feature.

This PR will only address minor styling changes for MVP release, next milestone (i.e. v1.2b/v1.3) will include actual layout changes.

Also updates the `help` command.

[You can try out the new UI here!](https://github.com/Polygonalr/tp/releases/tag/ui-v1.2)

Addresses #14

### CODE REVIEW

File: src/main/resources/view/HelpWindow.fxml [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
 <?import javafx.scene.Scene?>
 <?import javafx.scene.control.Button?>
-<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.Hyperlink?>
 <?import javafx.scene.image.Image?>
 <?import javafx.scene.layout.HBox?>
 <?import javafx.stage.Stage?>
 
-<fx:root resizable="false" title="Help" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
+<fx:root resizable="false" title="Help" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/18" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (malwaregarry) [inline src/main/resources/view/HelpWindow.fxml:12]: Scenebuilder might have changed this to 18 automatically. Should be version 11 just in case.

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 2]
```diff
@@ -63,49 +59,13 @@ public MainWindow(Stage primaryStage, Logic logic) {
...
-         * in CommandBox or ResultDisplay.
-         */
-        getRoot().addEventFilter(KeyEvent.KEY_PRESSED, event -> {
-            if (event.getTarget() instanceof TextInputControl && keyCombination.match(event)) {
-                menuItem.getOnAction().handle(new ActionEvent());
-                event.consume();
-            }
-        });
-    }
-
```
Comment (malwaregarry) [inline src/main/java/seedu/address/ui/MainWindow.java:108]: hmm, why was this removed?

**Review by malwaregarry — CHANGES_REQUESTED** [review]
LGTM just a minor problem I spotted.

**Comment by craeyeons** [comment]
LGTM

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 2 above)
Comment (craeyeons) [reply src/main/java/seedu/address/ui/MainWindow.java:108]: If anything, I think it's only line 86-106 that should be removed.

File: src/main/resources/view/MainWindow.fxml [hunk 3]
```diff
@@ -3,54 +3,66 @@
...
+               </left>
+               <right>
+                  <HBox alignment="TOP_RIGHT" prefHeight="100.0" prefWidth="200.0" BorderPane.alignment="CENTER">
+                     <children>
+                        <Button maxHeight="1.7976931348623157E308" mnemonicParsing="false" styleClass="menu-bar-nav-button" onAction="#handleHelp" text="Help" />
+                        <Button maxHeight="1.7976931348623157E308" mnemonicParsing="false" styleClass="menu-bar-nav-button" onAction="#handleExit" text="Exit" />
+                     </children>
+                  </HBox>
+               </right>
+            </BorderPane>
```
Comment (craeyeons) [inline src/main/resources/view/MainWindow.fxml:47]: Correct me if I'm wrong, but I think this chunk is indented more than it should be. I think it should be aligned with the deleted <MenuBar> section.

**Review by craeyeons — COMMENTED** [review]
I do not think that I quite understand JavaFx to begin with, but if it shows what it's supposed to, I think it's good enough. However, I think @malwaregarry 's queries need to be addressed first, especially on the part where setAccelerator() was removed.

Overall just some nitpicks from me that needs fixing.

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 2 above)
Comment (Polygonalr) [reply src/main/java/seedu/address/ui/MainWindow.java:108]: Yee it wasn't compatible with the new menu navbar I've implemented, I'll include the F1 hotkey again in a future commit.

File: src/main/resources/view/HelpWindow.fxml (hunk 1 above)
Comment (Polygonalr) [reply src/main/resources/view/HelpWindow.fxml:12]: Will fix this!

File: src/main/resources/view/MainWindow.fxml (hunk 3 above)
Comment (Polygonalr) [reply src/main/resources/view/MainWindow.fxml:47]: Alright, I'll take a look and get back to you later.

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 2 above)
Comment (malwaregarry) [reply src/main/java/seedu/address/ui/MainWindow.java:108]: oh ok that's not a big problem

Comment (Polygonalr) [reply src/main/java/seedu/address/ui/MainWindow.java]: @malwaregarry I've added back the F1 hotkey in a new function `setHelpShortcut` in the latest commit, don't think we need multiple layers of abstraction (i.e. `setAccelerators()` calling `setAccelerator(helpMenuItem, KeyCombination.valueOf("F1"));` in the old code) for now, unless we want to add more shortcuts in the future.

**Review by malwaregarry — APPROVED** [review]
LGTM

**Review by craeyeons — COMMENTED** [review]
LGTM!
